### PR TITLE
Close multiple simple issues in PSY

### LINKED ISSUE #1147: Adequacy check reports incorrect generation capacity value due to bug
The function adequacy_check in src/utils/IO/system_checks.jl currently reports an incorrect value for total generation capacity (this is a function that triggers a warning when there is less capacity than load). I believe this is due to a typo on line 126 of this file;  it should read components = get_components(component_type, sys). Line is here:
https://github.com/NREL-Sienna/PowerSystems.jl/blob/9e6f8a7e073ef7b606d3bc3a32640fcb69789a99/src/utils/IO/system_checks.jl#L126C37-L126C46

### LINKED ISSUE #1160: `System` constructor keyword argument `load_zone_formatter` is ignored
There is a kwarg `load_zone_formatter` listed as one of the supported `SYSTEM_KWARGS`:
https://github.com/NREL-Sienna/PowerSystems.jl/blob/71b9bf406ef7001eb5d6b56cc780b73093c9eb54/src/base.jl#L13
but, unlike all the other `SYSTEM_KWARGS`, nothing is ever done with it. This is the only reference to it in the codebase. I'm currently in a situation where I need to remap the names of load zones, so it would be nice to support this behavior as we do for buses, generators, branches, etc.

### LINKED ISSUE #1162: Add `show_time_series` for systems
Currently there is `show_time_series` for components, but no way to just get the Time Series Summary table that is printed when you print a `System`. This could redirect to `show_time_series_data` in infrastructure systems

### LINKED ISSUE #1165: Add per-end conductances to `Line`
The `Line` struct allows for the representation of one susceptance at the `from` end of the line and another at the `to` end:
https://github.com/NREL-Sienna/PowerSystems.jl/blob/71b9bf406ef7001eb5d6b56cc780b73093c9eb54/src/models/generated/Line.jl#L16. We should provide the same for conductance.

### LINKED ISSUE #1175: PowerModels Parser Overzealously Updates Bus Type
PowerSystems allows generators to be connected to PQ buses. So does the PSS/E data format, as far as I can tell. However, when PowerSystems parses such a thing from a PSS/E file, it converts the bus to PV:
https://github.com/NREL-Sienna/PowerSystems.jl/blob/c932c0b6bef52b911c5aedb1740a35fb620dae6e/src/parsers/pm_io/data.jl#L1698

Though probably intended as a data cleaning measure (quoth @claytonpbarrows), this strikes me as a bug as it leads to a Sienna system that does not correspond to the valid PSS/E system to the extent possible.

Note that the opposite case, which we also implement — converting a PV bus to PQ if there are no generators connected — is less problematic: the PSS/E spec states that PV buses must have at least one corresponding generator record, so PV buses without generators are invalid and we are free to do whatever we want with them.

I commented out the offending lines of `data.jl` and the PSY tests still passed, but I'd love to hear from others on the best thing to do here. Here's a minimal PSS/E RAW file that reproduces the issue (bus 1434):
```
0, 100.0, 33, 0, 1, 60.0
basic_2024
PSS/E 33.3 RAW via PowerFlows.jl, 2024-08-13T15:36:56.175
1434, 'B1434_S', 20.0, 1, 1, 10, , 1.0131474402609133, 21.436317553216863, 1.1, 0.9
4242, 'B4242_ND', 20.0, 3, 2, 2, , 1.02, 0.0, 1.1, 0.9
0
0
0
1434, 'OP', 520.0000000000001, 0.0, 3600.0, -3600.0, 1.0131474402609133, , 638.0, , , , , , 1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
Q
```

### PR DESCRIPTION
**Pull request by rodrigomha**
Closes #1175 via 692dfe05f7c7b154bd6ed5b7fb6c3bc32574e3be
Closes #1165 via 9dd3ef28969cb871f139c4546d41afb6dc87281d
Closes #1162 via a62741c08f7450bba5922c64e448101e8a3dda9a
Closes #1147 via 676e777428b11aca1dcfa6a0794d956d57944c7f
Closes #1160 via d954dee4abd771baa497f0ddd82e087e2a025597

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,6 @@ The `PowerSystems.jl` package provides a rigorous data model using Julia structu
 ## Version Advisory
 
 - PowerSystems will work with Julia v1.6+.
-- If you are planning to use `PowerSystems.jl` in your package, check the [roadmap to version 4.0](https://github.com/NREL-Sienna/PowerSystems.jl/projects/4) for upcoming changes
```
Comment (jd-lara) [inline README.md:16]: I need to make the version 5.0

**Review by jd-lara — APPROVED** [review]

File: src/parsers/pm_io/data.jl [hunk 2]
```diff
@@ -1695,10 +1695,7 @@ function correct_bus_types!(data::Dict{String, <:Any})
             end
 
             if bus_gens_count != 0 && bus["bus_type"] != 2
-                @info "active generators found at bus $(bus["bus_i"]), updating to bus type from $(bus["bus_type"]) to 2" maxlog =
-                    PS_MAX_LOG
-                bus["bus_type"] = 2
-                push!(modified, bus["index"])
+                @warn "active generators found at bus $(bus["bus_i"]) on bus type $(bus["bus_type"]), i.e. different than 2 (PV). Consider checking your data inputs."
```
Comment (GabrielKS) [inline src/parsers/pm_io/data.jl:1698]: Not all the PSB tests run in CI, right? Can we make sure that this isn't breaking anything in PSB?

File: src/parsers/power_models_data.jl [hunk 3]
```diff
@@ -346,9 +345,15 @@ function read_loadzones!(
         load_zone_map[zone]["pd"] += get(load, "py", 0.0)
         load_zone_map[zone]["qd"] += get(load, "qy", 0.0)
     end
+
+    default_loadzone_naming = x -> string(x)
```
Comment (GabrielKS) [inline src/parsers/power_models_data.jl:349]:
```suggestion
    default_loadzone_naming = string
```

File: src/parsers/power_models_data.jl [hunk 4]
```diff
@@ -346,9 +345,15 @@ function read_loadzones!(
...
+    # The formatter for loadzone_name should be a function that transform the LoadZone Int to a String
+    _get_name = get(kwargs, :loadzone_name_formatter, default_loadzone_naming)
```
Comment (GabrielKS) [inline src/parsers/power_models_data.jl:351]: Can we get unit tests for this?

**Review by GabrielKS — CHANGES_REQUESTED** [review]

File: src/parsers/power_models_data.jl (hunk 3 above)
Comment (jd-lara) [reply src/parsers/power_models_data.jl:349]: good catch

File: test/test_parse_psse.jl [hunk 5]
```diff
@@ -36,4 +36,15 @@ end
...
     @test get_available(get_component(ThermalStandard, sys, "generator-2438-EG")) == 1
+
+    @info "Testing Load Zone Formatter"
+    PSB.clear_serialized_systems("psse_Benchmark_4ger_33_2015_sys")
+    sys3 = build_system(
+        PSYTestSystems,
+        "psse_Benchmark_4ger_33_2015_sys";
+        loadzone_name_formatter = x -> string(3 * x),
+    )
+    lz_original = first(get_components(LoadZone, sys2))
```
Comment (GabrielKS) [inline test/test_parse_psse.jl:47]: I don't think you can safely assume that the result of `get_components` will be in the same order every time. Maybe get all the `LoadZone` names in each, sort those lists, and compare the lists?

**Review by GabrielKS — CHANGES_REQUESTED** [review]

File: test/test_parse_psse.jl (hunk 5 above)
Comment (rodrigomha) [reply test/test_parse_psse.jl:47]: There is only one load zone for that system, that's why I can use first

Comment (GabrielKS) [reply test/test_parse_psse.jl:47]: Oh okay

Comment (GabrielKS) [inline test/test_parse_psse.jl:47]: nitpick:
```suggestion
    lz_original = only(get_components(LoadZone, sys2))
```

File: test/test_parse_psse.jl [hunk 6]
```diff
@@ -36,4 +36,15 @@ end
...
+    lz_new = first(get_components(LoadZone, sys3))
```
Comment (GabrielKS) [inline test/test_parse_psse.jl:48]:
```suggestion
    lz_new = only(get_components(LoadZone, sys3))
```

**Review by GabrielKS — APPROVED** [review]
Looks good!

**Pull request merged**